lib/modules/common/constants.dart

```diff
@@ -2,4 +2,5 @@ class Paths {
   static const String HOME = '/';
   static const String LOGIN = '/login';
   static const String CONTRIBUTORS = '/contributors';
+  static const String SUMMARY_SCREEN = '/summary_screen';
```

Comment:
i did refactor the routes in a class... and i'll merge it soon, you'll have to solve some conflicts here.

Reply:
ok, no probs